docs/advanced/advanced_compatibility_reqs.md

```diff
@@ -0,0 +1,7 @@
+# Compatibility and Requirements
+
+For iSCSI single path users (RHEL only) verify that multipathing is installed and running.
```

Comment:
continuing [conversation](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r805321132):
@roysahar could we let the storage documentation cover the multipath conf part, as we did until now?